pkg/Microsoft.ML.DnnImageFeaturizer.AlexNet/Microsoft.ML.DnnImageFeaturizer.AlexNet.nupkgproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET component for pretrained AlexNet image featurization</PackageDescription>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.ML.OnnxTransform\Microsoft.ML.OnnxTransform.nupkgproj" />
+  </ItemGroup>
+
+  <ItemGroup>  
+    <Content Include="$(RepoRoot)\bin\obj\DnnImageModels\AlexNetOnnx\AlexNet.onnx" Pack="true" PackagePath="contentFiles\any\any\DnnImageModels\AlexNetOnnx;content\DnnImageModels\AlexNetOnnx" /> 
```

Comment:
>PackagePath="contentFiles\any\any\DnnImageModels\AlexNetOnnx;content\DnnImageModels\AlexNetOnnx" [](start = 95, length = 96)

We shouldn't be duplicating the files in the .nupkg. This will lead to larger disk usages, since the file will be duplicated.

Reply:
From what I was reading in the documentation, it seemed like depending on how the person uses/includes the NuGet, it might look for the files either in the content directory or in the contentFiles directory. I noticed, for example, that creating a simple test app through VS it did not seem to find the files if they were only in contentFiles. And searching online made it seem like there is no easy solution to not replicate them?

---
In reply to: [234754082](https://github.com/dotnet/machinelearning/pull/1447#discussion_r234754082) [](ancestors = 234754082)